cpp/test_cross_language_invocation.py

```diff
@@ -0,0 +1,21 @@
+import ray
```

Comment:
A little weird to add this test py file in "cpp" directory. Can we move it to "cpp/src/ray/test/cluster/"?